doc/connectivity/resting_state_networks.rst

```diff
@@ -85,3 +88,60 @@ each component separately.
    displayed on your computer might not match those of the documentation. For
    a fair representation, you should display all components and
    investigate which one resemble those displayed above.
+
+Beyond ICA : Dictionary learning
+================================
+
+Recent work has shown that dictionary learning based techniques.
+Dictionary learning in neuro-imaging seek to extract a few representative
+temporal elements along with their sparse spatial loadings, which constitutes
+good extracted maps, that are usually less noisy and better localized than ICA
+extracted maps.
+
+.. topic:: **References**
+
+   * Gael Varoquaux et al. `Multi-subject dictionary learning to segment an atlas of brain spontaneous activity <http://hal.inria.fr/inria-00588898/en/>`_,
+   Information Processing in Medical Imaging, 2011, pp. 562-573, Lecture Notes
+   in Computer Science,
+
+Applying DictLearning
+---------------------
+
+DictLearning is a ready-to-use class that is used just like CanICA. Sparsity of
+output map is controlled by a penalty parameter alpha (using a larger alpha
```

Comment:
It would be nice to make an indentity to parameter "alpha". It is mixed in the sentence.